dissect/target/plugins/os/windows/log/evt.py

```diff
@@ -1,7 +1,7 @@
 import fnmatch
 import re
 from pathlib import Path
-from typing import Any, BinaryIO, Generator, List, Optional
+from typing import Any, BinaryIO, Generator, Iterator, List, Optional
```

Comment:
```suggestion
from typing import Any, BinaryIO, Iterator
```
Can you change these to the modern variants too.